sdk/iot/core/doc/resources/iot_hub_flow.puml

```diff
@@ -0,0 +1,120 @@
+@startuml
+

```

Comment:
Were you planning on shipping these to customer?  If yes, then we should talk with Eric (if not) because we'll need to have folks maintain it when changes happen.

My thought is sample code is easier to follow and something that we can put gates up to check if it's screwed up.

Reply:
Yes: I do plan to have these maintained.

Reply:
I would love to get to a point were we maintain things like this

Reply:
Cool - if you guys are tracking this, I'm have less concern it'll become abandonware.  Thanks

Reply:
Looks like the latest doxygen can use puml. http://www.doxygen.nl/manual/commands.html#cmdstartuml